src/host.js

```diff
@@ -52,7 +53,7 @@ function Host(elementProvider, src, options) {
 	if (options.syncFont) {
 		this.use(syncFont);
 	}
-
```

Comment:
Can we make this an option that's `false` by default? Just worried that lots of other consumers who don't know anything about syncing CSS variables will upgrade and suddenly have some code running.

Reply:
yea